Makefile

```diff
@@ -338,6 +338,10 @@ delete-mgt-cluster-registry: $(CTLPTL) ## Deletes Kind-dev Cluster and the local
 	$(CTLPTL) delete cluster kind-$(INFRA_SHORT)
 	$(CTLPTL) delete registry $(INFRA_SHORT)-registry
 
+generate-hcloud-token:
+	@test $${TPS_TOKEN?Please set environment variable}
```

Comment:
I would use `./hack/ensure-env-variables.sh` like in the places in the Makefile.

Reply:
@guettli I updated it to invoke `./hack/ensure-env-variables.sh` for checking env variables.